docs/csharp/language-reference/keywords/modifiers.md

```diff
@@ -1,36 +0,0 @@
----
```

Comment:
This is a fairly high page view topic (2,739 in the last month), so just linking to a general list of keywords seems unhelpful. I'm wondering if a new topic similar to the Statement Categories topic would be helpful as an ultimate replacement.

Reply:
I agree @rpetrusha 

The long term target is [this section](https://gist.github.com/BillWagner/047cb97f6c0fe542d3a1299e994b2acd#-controlling-access) in the new TOC. I should make those updates in the next sprint or two.